app/api/v1/visits.rb

```diff
@@ -0,0 +1,63 @@
+module V1
+  class Visits < API
+    desc 'create a new visit'
+    # params do
+    #   requires :cabinet, type: String
+    #   requires :notes, type: String
+    #   requires :date, type: String
+    #   requires :diagnosis_ids, type: Array
+    # end
+
+    helpers do
+      def current_user_visits
+        Visit.where(user_id: current_user.id)
+      end
+
+      def find_visit_by_id(id)
+        current_user_visits.find(id)
+      rescue ActiveRecord::RecordNotFound => e
+        error!({ error_messages: e.message }, 422)
+      end
+    end
+    # rubocop:disable Metrics/BlockLength
+    resources :visits do
+      post do
+        visit = Visit.new(params)
+        visit.user = current_user
+
+        if visit.save
```

Comment:
`current_user.visits.create!(params)`

It's not critical, but handling this ☝🏻 in a few lines looks weird